### PR TITLE
add setup script

### PR DESCRIPTION
**Pull request by akash1810**
using Brewfile to manage dependencies

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cask "AdoptOpenJDK/openjdk/adoptopenjdk8"
+
+tap "guardian/devtools"
```
Comment (sihil) [inline Brewfile]: I think it auto taps when you use a cask or brew that is fully qualified.

File: script/setup [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```
Comment (sihil) [inline script/setup]: `set -e` is your friend here (and in all bash files from now until eternity)

File: Brewfile (hunk 1 above)
Comment (akash1810) [reply Brewfile]: Oh really?! That makes things so much easier!

File: script/setup (hunk 2 above)
Comment (akash1810) [reply script/setup]: oops!

**Pull request merged**